sbin/policy.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+PROG=${0##*/}
+
+set -euo pipefail -o noclobber
+shopt -s extglob
+
+function usage {
+	((${1:-1} > 0)) && exec 1>&2
+	cat <<EOF
+Usage: $PROG [-o FILE] [-A | -D] POLICY-CMD [ARGS [\\; ...]]
```

Comment:
over at tpm2-tools, there is a PR that potentially closes my issue https://github.com/tpm2-software/tpm2-tools/issues/2761 and makes it possible to use the policy programs without a real TPM: https://github.com/tpm2-software/tpm2-tools/pull/2762

until then, we should mention that this does talk to the TPM in the usage.

Reply:
Nice!  I'll note this.

Reply:
Note added.